### PR TITLE
3 get api

### PR DESCRIPTION
**Pull request by zajk9999**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -25,6 +26,17 @@ describe("/api/topics", () => {
...
-
 });
 
+describe("/api", () => {
+	it("GET: 200 - responds with an object with all available endpoints", () => {
+		return request(app)
+			.get("/api")
+			.expect(200)
+			.then(({ body }) => {
+				console.log(body)
```
Comment (nc-jo-watson) [inline __tests__/app.test.js:37]: Don't forget to remove `console.log`s before submitting a PR 😄 We don't want them in production code.

File: app.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
 const express = require("express")
 const { getTopics } = require("./controllers/nc-news-controllers")
 const app = express()
+const endpoints = require("./endpoints.json")
 
+app.get("/api", (request, response) => {
```
Comment (nc-jo-watson) [inline app.js:6]: Data shouldn't be imported into the app, so we need to extract this callback function as a controller 👀 If the data is imported into the app, it exists on the memory of the server all the time, even if a request isn't made to the endpoint that responds with it. This is inefficient for the performance of our server.

**Review by nc-jo-watson — COMMENTED** [review]
Nearly there Łukasz! You just need to extract the callback on the /api endpoint to it's own controller file.

File: app.js (hunk 2 above)
Comment (zajk9999) [reply app.js:6]: Make sense, will do!!

**Pull request merged**